features/no-disk-space.feature

```diff
@@ -0,0 +1,16 @@
+Feature: No Disk Space Overlay
+
+  Background:
+    Given I have completed the basic setup
+
+  Scenario: I set the required space to 1 KB
+    Given I set the required space to 1 KB
+    And I check the disk space
+    Then The No Disk Space overlay should be hidden
+
+  Scenario: I set the required space to 100 TB
```

Comment:
@daniloprates the scenario title should be more descriptive - e.g. "No disk space overlay is shown when there is not enough available disk space and hidden once more disk space becomes available"